Kernel/Devices/VMWareSVGADevice.h

```diff
@@ -0,0 +1,265 @@
+/*
+ * Copyright (c) 2021, the Serenity OS Developers
+ * Copyright (c) 2021, Alexander Richards <electrodeyt@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+
+
+#pragma once
+
+#include <AK/OwnPtr.h>
+#include <AK/Vector.h>
+#include <Kernel/Devices/BlockDevice.h>
+#include <Kernel/IO.h>
+#include <Kernel/Interrupts/IRQHandler.h>
+#include <Kernel/PCI/Device.h>
+
+namespace Kernel {
+class VMWareSVGADevice final : public BlockDevice
+    , public IRQHandler {
```

Comment:
I'd recommend making this inherit `PCI::Device` instead of `IRQHandler` directly so that you can access to the PCI address at any point.